docs/content/direct/common-setup-core-chart.md

```diff
@@ -37,12 +38,14 @@ helm upgrade --install ks-core oci://ghcr.io/kubestellar/kubestellar/core-chart
 
 Remember to include the option `--set "kubeflex-operator.isOpenShift=true"`, when deploying in an OpenShift cluster.
 
-## Obtain the kubeconfigs to access the KubeFlex Control Planes
+## Kubeconfig contexts for Control Planes
 
-After the KubeStellar Core has fully deployed, before changing the kubeconfig context, `kflex` CLI can be used to retrieve the kubeconfig of all the KubeFlex Control Planes and store them as contexts of the current kubeconfig:

```

Comment:
You can not store a kubeconfig as a context; that is a conceptual mismatch. What a context holds is less than a kubeconfig, it is only a single client configuration.